spec.md

```diff
@@ -92,39 +91,52 @@ The `Span` interface must have the following capabilities:
                 +--------+--------+     CHILD OF SPAN C), AS WELL
                 |        |        |     AS SPANS G, H, AND I.
             [Span G] [Span H] [Span I]
-```
+~~~
 
 
-### Tracer
+## Tracer
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
-- Provide some form of access to a SpanPropagator (whether that's through embedding, inheritance, containment, or something else is per-platform)
+- Start a new `Span`. There must be a way for the caller to specify a parent `Span`, an explicit start timestamp (other than "now"), and an initial set of `Span` tags. **(py: `start_span`, go: `StartSpanWithOptions`)**
+- Return an `Injector` instance given a format identifier, or nil/null if the format is not supported by the implementation (see [Injector and Extractor](#injector-and-extractor))
```

Comment:
thinking out loud.. "format identifier" isn't intuitive, and it takes some navigation to find what it is.. maybe in this case, use an example?  Ex. "Return an `Injector` instance given a format identifier, like ???, or nill..."